src/components/Composer.vue

```diff
@@ -200,6 +200,13 @@
 						}}
 					</ActionLink>
 				</Actions>
+				<button v-if="savingDraft === undefined"
```

Comment:
now you see *Discard draft* only when no draft has been saved yet, as soon as there is one the button vanishes, don't you?